### DISCUSSION TITLE
Rollup

### DISCUSSION THREAD

**Discussion by oetiker** [oetiker/SmokePing#290] [General]
# RollUp Release 

 - automated build testing @oetiker 
 - support for influxdb @mad-ady
 - AnotherDNS update @parseword
 - new matcher ConsecutiveLoss @rborgmaster
 - lots of spelling fixes @jsoref
 - update FPing commandline @lelutin
 - better handling of return codes from probe binaries @lelutin
 - manual page for smokeinfo @lelutin
 - drop SSHv1 and rsa1 from SSH.pm @Strykar
 - disable page refresh while zooming @n0rc
 - overflow scrollbar @bevhost
 - update url in zoom mode @dekenberg
 - allow relative time entry for cropper @dekenberg
 - fix config reload for slaves @stromnet
 - improved template layout @bevhost
 - fix edgetrigger functionality @oetiker
 - fping protocol parameter optional @AID
 - exploss matcher fixes @dfrli
 - interface parameter for fping probe @saaly182
 - pass 6th argument to edgetrigger external alerts @Ryan Rawdon
 - red background for 100% loss @dagelf
 - probe target in graph title @Daxtorim
 - make loss colors more distinctive @Daxtorim
 - calculate loss buckets based on number of 'pings' @Daxtorim

<hr /><em>This discussion was created from the release <a href='https://github.com/oetiker/SmokePing/releases/tag/2.8.0'>Rollup</a>.</em>

**Comment by KnoAll** [comment]
smokeping-2.8.1.tar.gz does not seem to be a complete file, getting an error that it is not a valid format, and it is only 563kb vs prior release where it was 6m+

**Reply by jsoref** [reply]
```sh
$ diff -qr smokeping-2.*|grep Only
Only in smokeping-2.7.3: PERL_MODULES
Only in smokeping-2.7.3: README
Only in smokeping-2.8.1: VERSION
Only in smokeping-2.8.1: cpanfile
Only in smokeping-2.8.1/lib/Smokeping/matchers: ConsecutiveLoss.pm
Only in smokeping-2.7.3/thirdparty: CPAN
Only in smokeping-2.8.1/thirdparty: cpanfile-5.30.snapshot
```

Offhand, it looks like the previous version included a cpan install...

**Reply by KnoAll** [reply]
Apologies, in my case I was pulling with an automated script from https://oss.oetiker.ch/smokeping/pub/ and the new 2.8.1 is not there yet. Changed to pull from here in the Git release and it is working.

**Reply by oetiker** [reply]
Oops missed that one ... I uploaded it to the pub folder too now.

**Reply by oetiker** [reply]
The size of the 2.8.x archive is much smaller because I switched from including all the sources of the dependent modules to a system based on Carton where the modules get installed from the internet. I included a list of module versions for perl 5.30 which is the version of perl shipping with ubuntu 20.04. If you build with a different version of perl, it will simply get the appropriate modules as specified in the cpanfile ... (and also create a snapshot in the process) which you can use the get consistant rebuilds over time ...
